test/DynamoCoreTests/NodeToCodeTest.cs

```diff
@@ -479,6 +479,12 @@ public void TestBasicNode2CodeWorkFlow1()
 
             Assert.IsTrue(expr1.ToString().StartsWith("a = 1;"));
             Assert.IsTrue(expr2.ToString().StartsWith("x = a;"));
+
+            var undo = new DynamoModel.UndoRedoCommand(DynamoModel.UndoRedoCommand.Operation.Undo);
+            CurrentDynamoModel.ExecuteCommand(undo);
+
+            Assert.IsTrue(CurrentDynamoModel.CurrentWorkspace.Connectors.Count() == 2);
+            Assert.IsTrue(CurrentDynamoModel.CurrentWorkspace.Nodes.Count == 3);
```

Comment:
Thanks for adding the test cases! One minor suggestion, instead of `Assert.IsTrue` which tells us *ah this condition is evaluated to false*, can we use `Assert.AreEqual(2, Connectors.Count())` that tells us *I'm expecting 2 but it was 1*? That will make the error more descriptive.